setup.py

```diff
@@ -76,9 +77,14 @@ def compute_cmake_args():
             cmake_args += ['-DCMAKE_INSTALL_PREFIX=' + _ctx.cxx_install_root]
 
         else:
-            cxxLibDir = os.path.abspath(
-                os.path.join(setuptools.__file__, "../../opentimelineio/cxx-libs")
-            )
+            if "--user" in sys.argv:
```

Comment:
I know this is far from being elegant, but it looks like setuptools is not providing the information in a clean way.

Reply:
do these slashes still work on windows?  or do we need to switch to an `os.path.sep.join` kind of thing?